src/elements/root/ui-router.scss

```diff
@@ -23,7 +23,7 @@ $transition-duration: 250ms; // Also update in ui-router
 			box-shadow: 0 0 128px rgba(0, 0, 0, 0.8);
 
 			.y-scroll {
-				min-height: 100vh;
+				min-height: 93vh;
```

Comment:
Same issue here as with the %  comment before. Prefer `calc` over VH units here.